News/Services/ArticleService.cs

```diff
@@ -0,0 +1,53 @@
+﻿using AutoMapper;
+using News.DataAccess.Repo;
+using News.Entities;
+using News.Entities.Models;
+using News.Services.ServicesInterface;
+using System.IO;
+
+namespace News.Services
+{
+    public class ArticleService : IArticleService
+    {
+        private readonly IArticleRepo _repo;
+        private readonly IMapper _mapper;
+        public ArticleService(IArticleRepo repo, IMapper mapper) 
+        {
+            _repo = repo;
+            _mapper = mapper;
+        }
+        public ArticleModel Add(ArticleModel model)
+        {
+            var entity = _mapper.Map<ArticleModel, ArticleEntity>(model);
+
+            _repo.Add(entity);
+            _repo.SaveChanges();
```

Comment:
The same question. In my opinion we should save changes inside repos.

Reply:
it's depend, lets discus it